### PR TITLE
Add year date automatically

### PR DESCRIPTION
**Pull request by rodrigovirgilio**
captura de tela 2018-10-29 14 58 58

### CODE REVIEW

File: public/assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var now = new Date()
+var year = now.getFullYear()
+
+document.querySelector('.footer-copyright span').
```
Comment (moiseshilario) [inline public/assets/js/index.js]: Let's change this a little bit, so we only change the year, and leave the rest of the string untouched.
SO we are going to create a new element on the footer-copyright, with class `copyright-year` for example

```suggestion
document.querySelector('.footer-copyright .copyright-year').innerHTML = year
```

File: public/assets/js/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var now = new Date()
```
Comment (andrerpbts) [inline public/assets/js/index.js]: What about?

```js
var year = new Date().getFullYear()
```

File: public/index.html [hunk 3]
```diff
@@ -223,18 +223,14 @@ <h1 class="ready-to-start">Ready to start?</h1>
               <a href="http://www.codeminer42.com/">
                 <img class="footer-cm42-logo" src="./assets/images/codeminer42-logo-white.png" alt="codeminer logo">
               </a>
-
               <div class="footer-copyright">
-                <span>2017 Codeminer 42</span>
+                <span></span>
```
Comment (moiseshilario) [inline public/index.html]: Here I suggest to put the text inside a `<p>` element like this

```html
<div class="footer-copyright">
  <p>
    <span class="copyright-year"></span>
    Codeminer 42
   </p>
</div>
```

File: public/assets/js/index.js (hunk 1 above)
Comment (moiseshilario) [reply public/assets/js/index.js]: Also, when you concatenate things, the dot (`.`) should be always on a new line
For example:
```js
document.querySelector('.element')
  .stuff1
  .stuff2
```

File: public/assets/js/index.js (hunk 2 above)
Comment (brunohkbx) [reply public/assets/js/index.js]: You should use `let` instead of `var`

File: public/index.html (hunk 3 above)
Comment (moiseshilario) [reply public/index.html]: And take care to HTML style not break. Hopefully not ;D

File: public/assets/js/index.js (hunk 1 above)
Comment (rodrigovirgilio) [reply public/assets/js/index.js]: 🤔 got it.

File: public/index.html (hunk 3 above)
Comment (rodrigovirgilio) [reply public/index.html]: broke not.
https://user-images.githubusercontent.com/392677/47670168-4e931800-db8b-11e8-971c-32eb5a3e5789.png
tks

File: public/assets/js/index.js (hunk 2 above)
Comment (moiseshilario) [reply public/assets/js/index.js]: We are not using babel. SO it's better to leave it `var` no harm here :)

Comment (rodrigovirgilio) [reply public/assets/js/index.js]: I returned the `var`

**Pull request merged**